Header       header
Aggregation  aggregation
CpuUsage[]   measurements
